.github/workflows/sync-upstream-pr.yml

```diff
@@ -0,0 +1,122 @@
+name: Sync with Upstream (create)
+
+on:
+  schedule:
+    - cron: "0 0 1 * *" # Runs at 00:00 on the first day of every month
+  workflow_dispatch:
+  # TODO: Remove this trigger once the workflow is ready
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.user.login != 'gha-runner-images-updater-test[bot]'
```

Comment:
Why should this be true? Not saying it's wrong just trying to understand.

Reply:
For now this needs to be true to prevent the PR workflow from running on bot PRs created by the workflow, since the workflow is temporarily running on PRs this would cause an infinite loop resulting in the bot creating 100s of PRs.